src/models/index.d.ts

```diff
@@ -7,8 +7,18 @@ interface IInvokeConfig {
 }
 
 interface IDBConfig {
-  params: { region: string; endpoint: string; convertEmptyValues: boolean };
-  table: string;
+  defects: {
```

Comment:
Hmm, im wondering could you not just reuse the defects object as it's literally the same?

Reply:
This is a representation of a config object, so needs both sections as they refer to separate tables (even though the table configs themselves have the same fields)